hepcrawl/utils.py

```diff
@@ -69,6 +71,28 @@ def collapse_initials(name):
     return name
 
 
+def split_fullname(author, surname="first"):
```

Comment:
Thanks for fixing this and adding the test. There is just one case that will cause this code to fail. In case, author is empty string or None.

And since `surname` is sort of a boolean flag, we can change it to be a boolean. E.g. `surname_first=True` as default argument. Then people would pass False if they want surname last.